packages/fields/types/CalendarDay/views/Cell/package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
is this due to `preconstruct`?

Reply:
Yes, though importantly, it's with a new approach to configuring entrypoints so all of these package.jsons were generated automatically(https://github.com/preconstruct/preconstruct/pull/275).

Reply:
I have nothing against it. I was thinking if this could go into `.preconstructrc` file which would lead to cleaner repo here.  Not sure if there is any plan to add such support (`.preconstruct.js`, `.preconstructrc.js{on}` etc.) to preconstruct.

Reply:
There are no plans whatsoever to have other places for configuration in Preconstruct. This file also isn't really _for_ Preconstruct, it's essentially an artifact of Preconstruct so that `@keystonejs/fields/types/CalendarDay/view/Cell` and etc. can be imported.

Reply:
I can not import `@keystonejs/fields/types/Integer/views/Controller`